src/elements/OcBreadcrumb.vue

```diff
@@ -26,6 +29,17 @@ export default {
       default: [],
       required: true,
     },
+    firstIsHome: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  methods: {
+    goHome() {
+      // We need the router here
+      //
+      // this.router.push('/')
```

Comment:
simply emit an event here?

Reply:
I could also leave it blank as we wouldn't need to catch that event anywhere else but within itself. Agree?

Reply:
how would a user of the component then get the event? Currently this is not usable